ParallaxGenLib/src/BethesdaDirectory.cpp

```diff
@@ -510,6 +510,7 @@ auto BethesdaDirectory::getBSALoadOrder() const -> vector<wstring> {
 }
 
 auto BethesdaDirectory::getPathLower(const filesystem::path &Path) -> filesystem::path {
+    // TODO UNICODE lower path
   return {boost::to_lower_copy(Path.wstring())};
```

Comment:
It seems like a combination of `PathCanonicalize()` and `_wcsicmp()` from `windows.h` could do the trick here. I have not been able to find any documentation on how windows itself does it

Reply:
not needed anymore since skyrim does not support non-ascii characters correctly for file names